content/en/blog/_posts/2022-08-04-cgroupv2-ga.md

```diff
@@ -0,0 +1,95 @@
+* The kubelet and the container runtime are configured to use the [systemd cgroup driver](/docs/setup/production-environment/container-runtimes#systemd-cgroup-driver)
+
+The kubelet and container runtime use a [cgroup driver](/docs/setup/production-environment/container-runtimes#cgroup-drivers) to set cgroup limits. When using cgroup v2, it's highly recommended that both kubelet and your container runtime (e.g. containerd or CRI-O) use [systemd cgroup driver](/docs/setup/production-environment/container-runtimes#systemd-cgroup-driver). This ensures that there is only a single cgroup manager on the system. Refer to the [systemd cgroup driver documentation](/docs/setup/production-environment/container-runtimes#systemd-cgroup-driver) about how to configure kubelet and the container runtime to use systemd as the cgroup driver.
+
+## Migrating to cgroup v2
+
+When you run kubelet on a Linux distribution that enables cgroup v2, kubelet should automatically perform accordingly without any additional configuration required.
+
+You should not find any noticeable difference in the user experience when switching to cgroup v2, unless users are accessing the cgroup file system directly, either on the node or from within the containers.
+
+It's important to note that the cgroup v2 uses a different API compared to cgroup v1, so if there any applications that read the cgroup file system, they should be updated to support cgroup v2. For example:
+
+* Some third-party monitoring and security agents may depend on the cgroup filesystem. Update these agents to versions that support cgroup v2.
+* If you run [cAdvisor](https://github.com/google/cadvisor) as a stand-alone DaemonSet for monitoring pods and containers, update it to v0.43.0 or later.
+* If you deploy Java applications with the JDK, prefer to use JDK 11.0.16 and later or JDK 15 and later, which [fully support cgroup v2](https://bugs.openjdk.org/browse/JDK-8230305).
+
+## How can I learn more?
+
+* Read the [Kubernetes cgroup v2 documentation](/docs/concepts/architecture/cgroups)
+* Read the enhancement proposal, [KEP 2254](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2254-cgroup-v2/README.md)
+* Learn more about [cgroups](https://man7.org/linux/man-pages/man7/cgroups.7.html) on Linux Man Pages and [cgroup v2](https://docs.kernel.org/admin-guide/cgroup-v2.html) on the Linux Kernel documentation
+
+
+## How do I get involved?
+
+Your feedback is always welcome! SIG Node meets regularly and can be reached via [Slack](https://slack.k8s.io/) (channel `#sig-node`), or the SIG's [mailing list](https://github.com/kubernetes/community/tree/master/sig-node#contact).
+We would love work on new resource management capabilities on top of cgroup v2.
+
+cgroup v2 has had a long journey and is a great example of OSS community collaboration across the industry since it requires work across the stack from the Linux Kernel to systemd to container runtimes, and Kubernetes.
```

Comment:
```suggestion
cgroup v2 has had a long journey and is a great example of open source community collaboration
across the industry because it required work across the stack, from the Linux Kernel to systemd to
various container runtimes, and (of course) Kubernetes.
```